modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/internal/v2_0/Capabilities200ServiceInfoOnlineTest.java

```diff
@@ -20,13 +20,15 @@
 import org.geotools.wfs.v2_0.WFSConfiguration;
 import org.geotools.xsd.Parser;
 import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
 
 /**
  * Test class for {@link Capabilities200ServiceInfo}.
  *
  * @author Matthias Schulze (LDBV at ldbv dot bayern dot de)
  */
+@Ignore
```

Comment:
This @Ignore  should at least have a reason  why it should be ignored eg. `@Ignore("service offline... or something")`

But I think you could update the `SERVER_URL` to https://laermkartierung.eisenbahn-bundesamt.de/deegree/services/wfs?service=WFS&request=GetCapabilities

Reply:
Tried that, but there are something with the certificate.

Reply:
@mprins Could you have a new look. I've changed to https, and added a reason for Ignore.
I also moved the test into "online" package.

Reply:
@roarbra are you sure you tried the link mprins provided? The certificate is invalid for laermkartierung1.eisenbahn-bundesamt.de (as used in the test) but valid for laermkartierung.eisenbahn-bundesamt.de (notice the missing 1 in the name).

Reply:
Thanks, I didn't notice. Now it's working.